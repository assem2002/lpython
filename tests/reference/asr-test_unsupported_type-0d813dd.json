{
    "basename": "asr-test_unsupported_type-0d813dd",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_unsupported_type.py",
    "infile_hash": "ff3af4fffe097c4ecd6ab673301b37806aefdd77c661d5b4afbbfc5d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_unsupported_type-0d813dd.stderr",
    "stderr_hash": "21fc45a6699d6b420540b4b862766fc022ac9978ceffc10c2b613c25",
    "returncode": 2
}