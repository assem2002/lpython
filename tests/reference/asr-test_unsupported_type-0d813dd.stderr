semantic error: The type 'i128' is undeclared.
 --> tests/errors/test_unsupported_type.py:2:8
  |
2 |     i: i128
  |        ^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
